src/ServiceManagement/StorSimple/Commands.StorSimple.Test/app.config

```diff
@@ -1,11 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@EmmaZhu This file is required for StorSimple tests (I know this is non-standard, I stumble over it myself, I suspect that if you revert this changes, the tests above will work.  Btw, it's important that this file is called 'App.config' and not 'app.config'